src/main/riddl/Test/messages.riddl

```diff
@@ -1,40 +1,116 @@
+} briefly "A reply from creating multiple organizations at once" explained by {
+| This event will contain all OrganizationIds created for the CreateOrganizations command it replies to
+}
+
+command CreateMembers is {
+  memberInfos: MembersContext.Info+,
+  creatingMember: MemberId
+} briefly "A command to create multiple members at once" explained by {
+| This command will create one organization for each of the Info values
+| provided. Organization can be specified via Info.organizationMembership.
 }
 
+event MembersCreated is {
+  membersCreated: MemberId+
+} briefly "A reply from creating multiple members at once" explained by {
+| This event will contain all MemberIds created for the CreateMembers command it replies to.
+}
+
+command CreateEvents is {
+  orgInfos: EventInfo+,
+  creatingMember: MemberId
+} briefly "A command to create multiple events at once" explained by {
+| This command will create one event for each of the Info values
+| provided. Organization can be provided via EventInfo.SponsoringOrg.
+}
+
+event EventCreated is {
+  eventsCreated: EventId+
+} briefly "A reply from creating multiple events at once" explained by {
+| This event will contain all EventIds created for the CreateEvents command it replies to.
+}
+
+command CreateStores is {
+  storeInfos: StoreInfo+,
+  creatingMember: MemberId
+} briefly "A command to create multiple stores at once" explained by {
+| This command will create one store for each of the Info values
+| provided.
+}
+
+event StoresCreated is {
+  storesCreated: StoreId+
+} briefly "A reply from creating multiple stores at once" explained by {
+| This event will contain all StoreIds created for the CreateStores command it replies to.
+}
+
+command CreateProducts is {
+  productInfos: ProductInfo+,
+  creatingMember: MemberId
+} briefly "A command to create multiple products at once" explained by {
+| This command will create one product for each of the Info values
+| provided. The ProductInfo includes a field productDetails, which can be used to choose ticket types.
+| ProductInfo required an EventId. StoreId is required in ProductInfo, as well as various metadata fields.
+}
+
+event ProductsCreated is {
+  productsCreated: SKU+
+} briefly "A reply from creating multiple products at once" explained by {
+| This event will contain all ProductIds created for the CreateProducts command it replies to.
+}
+
+command CreateOrders is {
+  orderInfos: OrderInfo+,
+  store: StoreId,
+  creatingMember: MemberId
+} briefly "A command to create multiple orders at once" explained by {
+| This command will create one order for each of the Info values
+| provided. The Info includes a lineItems field which can be used to reference product SKUs.
+}
+
+event OrdersCreated is {
+  orders: OrderId+
+} briefly "A reply from creating multiple orders at once" explained by {
+| This event will contain all OrderIds created for the CreateOrders command it replies to.
+}
+
+command CreateOrdersForMembers is {
```

Comment:
For now, I think all orders must belong to a Member. At some point, we will want to enable guest orders. But just to speed things up let's just say all orders must be placed by a member. Therefore I don't think we need a CreateOrdersForMembers command. CreateOrders will be sufficient.

Reply:
The distinction here is that we are specifying members for events, not just using `creatingMember` for all events. Are you saying you would prefer we make individual commands for each member we want to create events for?

Reply:
CreateMember is one command. Once a Member is created they can place an Order. You have a pair of commands - CreateOrder, and CreateOrdersForMembers. I don't see the reason for both. You have to have a valid member to create an order. The alternate is a Guest order, which would not require a member to place an order. So I am suggesting that we just go with CreateOrder for now (because it implies you have a valid Member). Later we will introduce CreateGuestOrder. I think the intent of a RIDDL mapping is zipping together two lists. I think you are looking at this like a map command where we have a list of Members and an (singular) order template. Using the template we create an order for each of the members in the list. Is that the intent?